pallet.el

```diff
@@ -237,6 +238,32 @@ use `pallet--package-archives-copy' if USE-COPY is true."
       (message "Pallet: unpacking %s" package-name)
       (pallet--unpack-one package-name))))
 
+;; Command to remove unrequested packages
+(defun pallet-auto-remove-packages ()
```

Comment:
Yeah, I like "prune" better because "sync" doesn't tell what is being synced with what (i.e. it could just as easily do what `pallet-init` does).